bin.js

```diff
@@ -165,13 +167,24 @@ function create (dir, argv) {
       lib.writeIcon(dir, done)
     },
     function (done) {
-      var message = 'The train has departed! :steam_locomotive::train::train::train:'
+      var message = '.'
```

Comment:
Isn't this _too_ simplistic? Something like "initial commit" or "initial choo boilerplate" might be better.

Reply:
heh, yeah nah it's what I use to mark all my root commits. Feel it has a nice minimal vibe to it. Just like `./` marks the root path, `.` marks the root commit. Or hidden commit because you're not the person making it.

Probably also not using any specific language might make folks from different backgrounds feel at home (e.g. non-English speakers). This allows folks to write their commit history themselves without it being tugged into any particular direction.

Reply:
I mean, this might be solved by our localization efforts in the future, but if it works for you it's cool by me!